hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -3873,12 +3873,12 @@ hazelcast:
   #      used by map stores and jdbc sinks and sources.
   data-link:
     mysql-database:
-      class-name: com.hazelcast.datalink.JdbcDataLink
+      type: type
```

Comment:
minor: maybe change the actual type name to `jdbc`?

Reply:
2463c5d1464